src/prefect/serialization/schedule.py

```diff
@@ -133,19 +131,3 @@ class Meta:
             valid_functions=ADJUSTMENTS, reject_invalid=True, allow_none=True
         )
     )
-
-
-class ScheduleSchema(OneOfSchema):
-    """
-    Field that chooses between several nested schemas. This class is preserved for pre-0.6.1
-    compatibility, and is deprecated in favor of NewScheduleSchema.
-    """
-
-    # map class name to schema
-    type_schemas = {
-        "Schedule": NewScheduleSchema,
-        "IntervalSchedule": schedule_compat.IntervalScheduleSchema,
```

Comment:
do we need any tests for serializing and deserializing `Intervalschedule` and `CronSchedule`?

it seems like since both are just functions returning `Schedule` classes we're okay

Reply:
I think we will just let them fail to deserialize since they're for pre 0.6.1 support